include-blang/cooperative_groups.h

```diff
@@ -0,0 +1,32 @@
+#ifndef _COOPERATIVE_GROUPS_H_
+#define _COOPERATIVE_GROUPS_H_
+
+namespace cooperative_groups {
+  class grid_group {
+    public:
+      __forceinline__ __device__ void sync() const {
```

Comment:
Elsewhere we use `__inline__`.
```suggestion
      __inline__ __device__ void sync() const {
```
Similar for the other functions with bodies below.

Reply:
I will make this change at all places and update this PR.